sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -166,7 +182,8 @@ object ScalaNativePluginInternal {
     scalaNativeConfigSettings ++ Seq(
       test := (test in NativeTest).value,
       testOnly := (testOnly in NativeTest).evaluated,
-      testQuick := (testQuick in NativeTest).evaluated
+      testQuick := (testQuick in NativeTest).evaluated,
+      nativeLink := (nativeLink in NativeTest).value
```

Comment:
Why is this addition necessary?

Reply:
This makes it possible to run `test:nativeLink` for quickly testing LLVM compilation without actually running. It's especially useful when testing on macOS where running the container can get pretty slow.